### DISCUSSION TITLE
Load config  no-binary from pyproject.toml

### DISCUSSION THREAD

**Discussion by Pajinek** [python-poetry/poetry#8406] [General]
Hello, 
I wanted to set no-binary package for in installer in pyproject.toml, but this filed is not loaded from this file and nothing happened. I figured out that I need to have this field `no-binary =...` in separated file poetry.toml.

```
[installer]
no-binary = ["kiwi-token"]
```

Is config possible to set in pyproject.toml as well? I tried to set this configuration in pyproject.tml and all of following configs doesnt work.

```
[tool.poetry.installer] 
no-binary = ["kiwi-token"]
```

```
[installer] 
no-binary = ["kiwi-token"]
```

```
[tool.installer] 
no-binary = ["kiwi-token"]
```

**Comment by radoering** [comment]
no, see https://python-poetry.org/docs/configuration

**Comment by Pajinek** [comment]
@radoering I read docs, but I am not sure. thanks.

```Your local configuration of Poetry application is stored in the poetry.toml file, which is separate from pyproject.toml.```
